python/pyarrow/plasma.pyx

```diff
@@ -370,7 +370,8 @@ cdef class PlasmaClient:
                                          object_buffers[i].metadata_size))
         return result
 
-    def put(self, object value, ObjectID object_id=None):
+    def put(self, object value, ObjectID object_id=None,
+            serialization_context=None):
```

Comment:
Could you the type to the new parameter?

Reply:
I believe it can be any Python object having the required methods

Reply:
Yes, this is true!